examples/simpleReadEvery/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/xeoncross/wav"
```

Comment:
Thank you, this was the missing piece in my workflow. I have indeed been bitten by duplicate projects and manually adding remotes to the parent repo is a much better solution.